hieradata/nodes/dev-anthrax.yaml

```diff
@@ -0,0 +1 @@
+ocf::packages::postfix::manage: false
```

Comment:
That's a good point, I hadn't thought about the parse order issues there. This would be fine for now then, a little duplication doesn't hurt. Would be something to think about, maybe having a role pattern would be the way to go then...